SpecialRequestWiki.php

```diff
@@ -82,10 +82,14 @@ protected function getFormFields() {
 
 	public function onSubmit( array $formData ) {
 		$dbname = $formData['subdomain'] . 'wiki';
-		$private = is_null( $formData['private'] ) ? O : 1;
 		$url = $formData['subdomain'] . ".miraheze.org";
 
 		$request = $this->getRequest();
+		
+		$params['private'] = $request->getVal ('private');
```

Comment:
Why are you rewriting form data?

Reply:
We are discussing it on irc, but private seems to be the default when requesting new wiki's according to void.